arcgis-ios-sdk-samples/Content Display Logic/Controllers/OptionsTableViewController.swift

```diff
@@ -31,15 +31,16 @@ class OptionsTableViewController: UITableViewController {
     /// The index of the currently selected option or `nil` if no option is
     /// selected.
     private var selectedIndex: Int?
-    private let onChange: (Int) -> Void
+    private let onChange: (Int?) -> Void
+    var allowsEmptySelection = false
     
     /// Creates a new instance with the options, selected index, and selection
     /// change handler.
     /// - Parameters:
     ///   - options: The options displayed by the view controller.
     ///   - selectedIndex: The index of the currently selected option or `nil`.
     ///   - onChange: A closure called when the selected option has changed.
-    init(options: [Option], selectedIndex: Int?, onChange: @escaping (Int) -> Void) {
+    init(options: [Option], selectedIndex: Int?, onChange: @escaping (Int?) -> Void) {
```

Comment:
You and Phil might have discussed here and came up with the instance property solution. I think that is fine, but I prefer to put it in the initializer as a parameter with default value like this

```swift
init(options: [Option], selectedIndex: Int?, allowsEmptySelection: Bool = false, onChange: @escaping (Int?) -> Void) { // ... }
```

so that in `Create and save the map` we don't need to set it on another line.

Note if you gonna apply this change, be sure to also update the comment docstrings for both `init` methods, and make the `allowsEmptySelection` property private.